src/redux/rockets/rocket.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable no-case-declarations */
```

Comment:
- [ ] As already pointed out by the previous reviewer, please kindly remove this line and avoid disabling linters.